src/views/core/bodies/ChangeBoardModal.vue

```diff
@@ -0,0 +1,208 @@
+          </template>
+        </b-field>
+
+        <b-autocomplete
+          v-model="position.name"
+          :data="members"
+          :loading="isLoading"
+          @input="query => fetchMembers(query)"
+          @select="user => position.id = user.id"
+          :expanded="true"
+          field="name">
+          <template slot-scope="props">
+            <div class="media">
+              <div class="media-content">{{ props.option.first_name }} {{ props.option.last_name }}</div>
+            </div>
+          </template>
+        </b-autocomplete>
+
+        <b-button v-if="!position.required" class="button is-danger" @click="deletePosition(index)">
+          <span class="white"><font-awesome-icon :icon="['fa', 'trash-alt']" /></span>
+        </b-button>
+      </b-field>
+
+      <b-field>
+        <button class="button is-primary" @click="addPosition()">Add new position</button>
+      </b-field>
+
+      <hr>
+
+      <div class="field">
+        <label class="label">Message</label>
+        <div class="control">
+          <b-input type="textarea" v-model="board.message" />
+        </div>
+        <label class="label">Preview <MarkdownTooltip/></label>
+        <div class="content">
+          <span v-html="$options.filters.markdown(board.message)" />
+        </div>
+      </div>
+    </section>
+
+    <footer class="modal-card-foot">
+      <button class="button is-primary" @click="saveChangeBoard()">Save changes</button>
+      <button class="button" @click="$parent.close()">Cancel</button>
+    </footer>
+
+  </div>
+</template>
+
+<script>
+import MarkdownTooltip from '../../../components/tooltips/MarkdownTooltip'
+
+export default {
+  components: {
+    MarkdownTooltip
+  },
+  name: 'ChangeBoardModal',
+  props: ['body', 'services', 'showError', 'showSuccess'],
+  data () {
+    return {
+      board: {
+        elected_date: '',
+        start_date: '',
+        end_date: null,
+        positions: [
+          { function: 'President', name: '', id: '', required: true },
+          { function: 'Secretary', name: '', id: '', required: true },
+          { function: 'Treasurer', name: '', id: '', required: true }
+        ],
+        message: ''
+      },
+      members: [],
+      isLoading: false,
+      isSaving: false
+    }
+  },
+  methods: {
+    saveChangeBoard () {
+      this.isSaving = true
+
+      const board_export = {
+        body_id: this.body.id,
+        elected_date: this.board.elected_date,
+        start_date: this.board.start_date,
+        end_date: this.board.end_date,
+        president: this.board.positions[0].id,
+        secretary: this.board.positions[1].id,
+        treasurer: this.board.positions[2].id,
+        // TODO: Think of a good way to export other positions
+        // Probably just with function and id
+        // other_members: {
+        //
+        // },
+        message: this.board.message
+        // TODO: Add image
```

Comment:
If this works, maybe we could do the same at event creation?

Reply:
You mean with the image uploading and stuff?

Reply:
Yes